activesupport/lib/active_support/rails.rb

```diff
@@ -1,17 +1,15 @@
-# This is private interface.
```

Comment:
Not sure if this is correct :)

If I understand this file correctly, it's to setup common Rails requires across components. Not something for apps to piggy back on; they've got the top level `'active_support'` require 😁